src/SFA.DAS.ApprenticeCommitments.Database/Tables/ApprenticeEmailHistory.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE TABLE [dbo].[ApprenticeEmailHistory] (
+    [ChangedOn]    DATETIME2 (7)  NOT NULL,
+    [ApprenticeId] BIGINT         NOT NULL,
+    [Id]           INT            IDENTITY (1, 1) NOT NULL,
```

Comment:
I have this tiny habit of starting these ids at a offset from 1. (like 1000. 2000, or 10000 .....) The only logic for this is if we end up joining using the wrong id, we usually spot it immediately (as the join won't find a record).

Also Id fields tend to be added as BIGINT in our apps